tools/smtpmock/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # build stage
-FROM golang:1.15 AS builder
+FROM golang:1.17 AS builder
```

Comment:
> ```
> # github.com/ovh/cds/tools/smtpmock
> ./client.go:49:14: undefined: io.ReadAll
> ```
@yesnault This should be fixed now. The SMTPMock Dockerfile is using Go 1.15 instead of 1.17.

Reply:
@Juneezee  yes, thank you. Can you re-run go mod tidy in tools/smtpmock please? seems that go.mod is no up to date

Reply:
@yesnault Done. Please take a look again. Thank you.

Reply:
@Juneezee  It's now ok, thank you!